index.js

```diff
@@ -1,5 +1,6 @@
 // Addons
 import Confirm from 'src/addons/Confirm/Confirm';
+import Image from 'src/addons/Image/Image';
```

Comment:
Semantic UI Image is an `element`: http://semantic-ui.com/elements/image.html

The `addon` type is for components that Stardust adds to Semantic, like `Confirm`.  There is no `Confirm` component in Semantic UI so it is an `addon`.